fth/floats.fth

```diff
@@ -137,7 +144,7 @@ fvariable FVAR-REP  \ scratch var for represent
         0 -> n
     ELSE
         fvar-rep f@
-        flog
+        safe-flog
```

Comment:
I believe that REPRESENT is already safe because we check for F0< on line 132 and F0= at line 140.